tests/test_openapi.py

```diff
@@ -312,3 +314,45 @@ def endpoint_test(form: BaseRequest):
             },
             "required": True
         }
+
+
+class Detail(BaseModel):
+    num: int
+
+
+T = TypeVar("T", bound=BaseModel)
+
+
+class GenericResponse(GenericModel, Generic[T]):
+    detail: T
+
+
+class ListGenericResponse(GenericModel, Generic[T]):
+    items: List[GenericResponse[T]]
```

Comment:
I used `list` here last time, but in python 3.7 etc. it has to be `List`, so I modified it